### ISSUE TITLE
Escape characters in path

### ISSUE THREAD

**Issue by ClintH** [RafidMuhymin/astro-imagetools#51]
Platform: Windows
astro-imagetools: 0.6.3
astro: 1.0.0-beta.27
node: 18.1.0
npm: 8.8.0

Hi! 

In an .astro file front-matter, I have:
`import test from '/src/images/test.jpg';`

and below in the HTML:
`<img src={test}>`

When running with npm run dev, the IMG src is set to `/assets    est@2880w.32f1e65e.jpeg` (ie. a tab character instead of the string `/t`).

Doing a build produces the src `//assets    est@2880w.32f1e65e.jpeg`. 

Same issue, but now with an extra forward slash for some reason.

**Comment by alextim** [comment]
Platform: Windows  
astro-imagetools: 0.6.3  
astro: 1.0.0-beta.27  
node: 18.1.0  
pnpm: 7.0.0  

I have the same issue with the same input data. 
`Img` & `Picture` components have the similar behavior.  


But when I changed `src` from local file to external url

```js
	<Img src="https://picsum.photos/1024/768/" alt="test" />
```
I got an error in the line 35 of `api/utils/getSrcset.js`

```js
const srcset = (await import(id)).default;
```

```text
 error   Parse failure: Octal literal in strict mode (1:24)
  Contents of line 1: export default "/\assets\768.a1dab434b30a6983814ca2edf3430821@320w.ebdb3f89.jpeg 320w, /\assets\768.a1dab434b30a6983814ca2edf3430821@848w.ca7a5b0e.jpeg 848w, /\assets\768.a1dab434b30a6983814ca2edf3430821@1200w.4815d5aa.jpeg 1200w, /\assets\768.a1dab434b30a6983814ca2edf3430821@1024w.de9cd584.jpeg 1024w"        
    at ssrTransform (C:\Users\alex\Documents\Projects\astro-imagetools-test\node_modules\.pnpm\vite@2.9.8\node_modules\vite\dist\node\chunks\dep-e1fc1d62.js:54541:15)
    at doTransform (C:\Users\alex\Documents\Projects\astro-imagetools-test\node_modules\.pnpm\vite@2.9.8\node_modules\vite\dist\node\chunks\dep-e1fc1d62.js:55940:17)
src= /node_modules/.cache/astro-imagetools/768.a1dab434b30a6983814ca2edf3430821.jpg
id= /node_modules/.cache/astro-imagetools/768.a1dab434b30a6983814ca2edf3430821.jpg?format=jpeg&w=320;848;1200;1024&aspect=1.3333333333333333
```

**Comment by TimWeidner** [comment]
I too think there is a big error with unescaped characters here.
![image](https://user-images.githubusercontent.com/55436069/168174691-af46af2e-963a-40ef-8b46-c99b31708af6.png)

**Comment by RafidMuhymin** [comment]
Closing this issue as a fix for this has been released in `astro-imagetools@0.6.4`. Feel free to reopen this issue if you are facing any issues.
